rp2040-hal-macros/README.md

```diff
@@ -10,18 +10,12 @@ Currently, it just unlocks all spinlocks before calling the entry function.
 
 # License
 
-Licensed under either of
-
-- Apache License, Version 2.0 (`APACHE2.0` or
-  http://www.apache.org/licenses/LICENSE-2.0)
-
-- MIT license (`MIT` or http://opensource.org/licenses/MIT)
-
-at your option.
-
-## Contribution
-
-Unless you explicitly state otherwise, any contribution intentionally submitted
-for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
-dual licensed as above, without any additional terms or conditions.
+The contents of this repository are dual-licensed under the _MIT OR Apache 2.0_
+License. That means you can choose either the MIT license or the Apache 2.0
+license when you re-use this code. See [`LICENSE-MIT`](./LICENSE-MIT) or
+[`LICENSE-APACHE`](./LICENSE-APACHE) for more information on each specific
+license. Our Apache 2.0 notices can be found in [`NOTICE`](./NOTICE).
+
+Any submissions to this project (e.g. as Pull Requests) must be made available
```

Comment:
IANAL, but I think the old wording regarding contributions was better.

Reply:
We had two kinds - do you mean the one with the `# Contributing` header? I can use that.